rubygem-apipie-rails/rubygem-apipie-rails.spec

```diff
@@ -84,6 +84,9 @@ cp -a .%{gem_dir}/* \
 %{gem_instdir}/%{gem_name}.gemspec
 
 %changelog
+* Wed Dec 06 2017 Patrick Creech <pcreech@redhat.com> - 0.5.6-1
+- Update apipie-rails to 0.5.6
```

Comment:
You can leave this part out, our process involves tito tagging after we merge which will generate the changelog.

Reply:
Removed